pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -2389,7 +2389,19 @@ func validateDisks(field *k8sfield.Path, disks []v1.Disk) []metav1.StatusCause {
 
 // Rejects kernel boot defined with initrd/kernel path but without an image
 func validateKernelBoot(field *k8sfield.Path, kernelBoot *v1.KernelBoot) (causes []metav1.StatusCause) {
-	if kernelBoot == nil || kernelBoot.Container == nil {
+	if kernelBoot == nil {
+		return
+	}
+
+	if kernelBoot.Container == nil {
```

Comment:
You could merge 2 ifs

Reply:
That would cause a panic (nil dereference at line 2410), `kernelBoot.Container == nil` is a valid condition.

Reply:
Exactly.
@xpivarc Do you agree or am I missing anything else?

Reply:
Right, I saw `}` as the last thing and thought its the end of the function... (Nothing wrong here, just my eyes :) )

Reply:
:)
Thanks!